share/picongpu/dockerfiles/README.rst

```diff
@@ -56,7 +56,7 @@ You can also push the result to dockerhub and singularity-hub (you need an accou
     cd ubuntu-1604
 
     # docker image
-    docker build -t ax3l/picongpu:0.5.0 .
```

Comment:
This change is wrong and introduced in https://github.com/ComputationalRadiationPhysics/picongpu/commit/0cc2a2291813e1012b4a10e8f029ce5b80cba6ca#diff-821c08a5fdb3c6cb5a0ea2a56bd807feeb65562430d68ae637a6b2d92f6db03e

I will fix it after this PR is merged!

Reply:
Okay